app/assets/stylesheets/_notes.scss

```diff
@@ -71,6 +72,39 @@
       border-color: transparent;
       width: 100%;
       position: relative;
+      height: 28px;
+    }
+
+    .search-options {
+      margin-top: 10px;
+
+      align-self: center;
+      display: flex;
+      flex-direction: row;
+      align-items: baseline;
+      gap: 0.25rem;
+      font-size: var(--sn-stylekit-font-size-p);
+      white-space: nowrap;
+
+      .bubble {
+        cursor: pointer;
+        padding: 2px 5px;
+
+        transition-property: color, background-color, border-color, transform;
+        transition-timing-function: cubic-bezier(.4,0,.2,1);
+        transition-duration: 300ms;
```

Comment:
Looking at the video, I think this transition is a bit slow. Something like `100ms` might look better, what do you think?

Reply:
Agree! Thank you for the suggestion. :grin: